### PR TITLE
Legacy css  carolyn  how to tab  issue556

### PR DESCRIPTION
**Pull request by carolyn-wang**
**Branch for Issue #556 (UI updates for How-To page)**
Removed this task from my previous pull request (Legacy css Carolyn issues 551 554 556) and moved it here instead

### CODE REVIEW

File: components/blocks/how-to-tabs/how-to-tab/how-to-tab.hbs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <div class="open">hide</div> --}}
+        <span>show</span>
+      </div>
+    </div>
+  </{{#if js}}button{{else}}label{{/if}}>
+  <div class="ht-tab-c">
+    {{{ tab.content }}}
+  </div>
+</div>
+{{/each}}
```
Comment (phillipbentonkelly) [inline components/blocks/how-to-tabs/how-to-tab/how-to-tab.hbs]: Check how you're ending these files, I see its yelling about 'no end of file' but its not doing it for others that end the same

**Review by phillipbentonkelly — APPROVED** [review]
LGTM

**Review by phillipbentonkelly — CHANGES_REQUESTED** [review]
- Un-delete `tabs` (components/base/tabs/)
- Checkout package-lock.json
- Fix merge conflicts

**Review by subaha-cob — APPROVED** [review]
LGTM

**Pull request merged**